test/util/e2e.go

```diff
@@ -112,3 +115,33 @@ func WaitForKubeFlowAvailability(ctx context.Context, k8sClient client.Client) {
 	kftoKey := types.NamespacedName{Namespace: "kubeflow", Name: "training-operator"}
 	waitForOperatorAvailability(ctx, k8sClient, kftoKey)
 }
+
+func GetE2eTestSleepImage() string {
+	E2eTestSleepImage := "gcr.io/k8s-staging-perf-tests/sleep:v0.1.0"
+	// Use e2e test image with digest if USE_E2E_TEST_IMAGE_DIGEST environment variable is provided and its value set to 'true', otherwise use default e2e test image with version tag
+	if useE2eTestImage, ok := os.LookupEnv("USE_E2E_TEST_IMAGE_DIGEST"); ok && useE2eTestImage == "true" {
+		e2eTestImageWithDigest, err := GetImageWithDigest(E2eTestSleepImage)
+		if err != nil {
+			fmt.Printf("unable to get image digest: %s", err)
+		}
+		return e2eTestImageWithDigest
+	}
+	return E2eTestSleepImage
+}
+
+func GetImageWithDigest(image string) (string, error) {
```

Comment:
Querying the digest defies the security advantages of the digests.

Why not just hard code the digest for the default image?

Reply:
The security reason is definitely something we need to take into account.

Moreover, we need to exactly know which exact images are used so that we can use the code in a disconnected env.